libraries/angular/package.json

```diff
@@ -19,17 +19,17 @@
     "karma-sourcemap-loader": "0.4.0",
     "karma-webpack": "5.0.0",
     "ts-helpers": "1.1.2",
-    "typescript": "3.9.10",
+    "typescript": "^3",
```

Comment:
can you keep this ~3.9 because of how TypeScript does semver? I know it's equivalent in the .9 case because they don't do .10 releases, but it's easier to stay consistent, imo